### ISSUE TITLE
Radicale proxy issue about anonymous access: Thunderbird caldav mod doesn't work

### ISSUE THREAD

**Issue by inkhey** [tracim/tracim#1573]
## Bug description & symptom

Thunderbird doesn't allow to be set up with "caldav" parameter (ics file work).

Radicale itself return error:
```
17:14:27 INFO  [radicale:409][waitress] PROPFIND request for '/calendar/workspace/2/' with depth '0' received from 127.0.0.1 using 'Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Thunderbird/60.3.0 Lightning/6.2.3'
17:14:27 INFO  [radicale:508][waitress] Access to '/calendar/workspace/2/' denied for anonymous user
17:14:27 INFO  [radicale:387][waitress] PROPFIND response status for '/calendar/workspace/2/' with depth '0' in 0.003 seconds: 401 Unauthorized
```

## Diagnostic

Currently,proxy used to radicale drop header authentification. This is a normal behaviour for a proxy, but radicale for some request case want an authentification header despite auth mecanism of radicale is set to None (so no check a all validity of user password here).

## Expected

- Do not drop header authentification in order to let radicale deal with a real auth header. We may replace it with a custom dumb auth header.

**Comment by inkhey** [comment]
https://github.com/tracim/tracim/pull/1574 fix was verified to work (Read/Write case) on : 
- thunderbird (behaviour when reader and try to delete event is error message popup)
- caldav sync (behaviour when reader and try to delete event is false positive, content not truly deleted but client think it is).

**Comment by PhilippeAccorsi** [comment]
Tested with OpenSync on Android and its work good.

**Comment by PhilippeAccorsi** [comment]
Some test with user with role reader and try to delete event:

With thunderbird error appear if I tried to delete event "modification_failed"
With OpenSync no error appear but after synchronize event is come back
With davdroid with this new name https://www.davx5.com/  (also available in F-droid )no error appear but after synchronize event is come back

With this other client, we have crazy situation:
With CalDAV-sync no error appear and after synchronize event is not visible on phone but visible in other support.
With CalDAV Sync Adapter no error appear and after synchronize event is not visible on phone but visible in other support.

With this two app, its not possible to see again the event we have tried to delete.
